docs/Packaging.md

```diff
@@ -8,8 +8,8 @@ are as follows:
 
 Klipper uses a C module to handle some kinematics calculations more quickly.
 This module needs to be compiled at packaging time to avoid introducing a
-runtime dependency on a compiler. To compile the C module, run `python2
-klippy/chelper/__init__.py`.
+runtime dependency on a compiler. To compile the C module, run
+`~/klippy-env/bin/python klippy/chelper/__init__.py`.
```

Comment:
self-review:

Problem this submission fixes:
Instruction to use `python2`, creates numerous stumling blocks for users following this documentation, primarily, that most folks no longer have a working python2 interperter on their system.
The change here, to simply use the python which should have been set up already, makes this documentation run correctly regardless of which python the user has installed- as well as more likely to contain the libraries required by the code which is run.

Most UNIX systems don't ship with python2 any more, (and it's getting harder to maintain the python2 ecosystem), so this line trips up anyone trying to follow the instructions.

This small submission is high impact to the audience of users who are following this documentation.